src/client/components/pages/contribute.js

```diff
@@ -15,11 +15,16 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
+/* eslint valid-jsdoc: "error" */
 
 const React = require('react');
 
 const PageHeader = require('react-bootstrap').PageHeader;
 
+/**
+* Renders the the document and display the 'Contribute' Page.
+* @returns {ReactElement} a HTML document which displays the contribute page
```

Comment:
"the Contribute page" - you've capitalised elsewhere, so it makes sense to be consistent